authutils/auth_service_test.go

```diff
@@ -12,10 +13,10 @@ func TestCompareWithLatestBlockNumber(t *testing.T) {
 	config.Vip().Set(config.EthereumJsonRpcEndpointKey, "https://ropsten.infura.io")
 	currentBlockNum, _ := CurrentBlock()
 	err := CompareWithLatestBlockNumber(currentBlockNum.Add(currentBlockNum, big.NewInt(13)))
-	assert.Equal(t, "difference between the latest block chain number and the block number passed is 13 ", err.Error())
+	assert2.NotNil(t, err)
```

Comment:
I like previous version which checks message more.

Reply:
as it checks specific error not generic one

Reply:
Reverted back to the previous check on exact error message